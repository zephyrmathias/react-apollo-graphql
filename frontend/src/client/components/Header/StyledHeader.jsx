import styled from 'styled-components';

const StyledHeader = styled.div`
  background: pink;
`;

export default StyledHeader;
