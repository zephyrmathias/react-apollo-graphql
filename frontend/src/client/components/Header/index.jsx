import React from 'react';
import { NavLink } from 'react-router-dom';
import StyledHeader from './StyledHeader';

const Header = () => (
  <StyledHeader>
    <NavLink to="/">
      Home
    </NavLink>
    <NavLink to="/profile">
      Profile
    </NavLink>
  </StyledHeader>
);

export default Header;
