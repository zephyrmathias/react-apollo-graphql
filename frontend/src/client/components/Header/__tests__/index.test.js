import React from 'react';
import { shallow } from 'enzyme';
import Header from '../index';

describe('components: <Header />', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
