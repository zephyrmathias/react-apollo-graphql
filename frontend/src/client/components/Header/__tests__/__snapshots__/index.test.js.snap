// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components: <Header /> should render correctly 1`] = `
<styled.div>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/"
  >
    Home
  </NavLink>
</styled.div>
`;
